packages/eas-cli/package.json

```diff
@@ -30,6 +30,7 @@
     "@oclif/core": "1.9.0",
     "@urql/core": "2.4.4",
     "@urql/exchange-retry": "0.3.3",
+    "ajv": "^6.12.6",
```

Comment:
we already use joi for validation, do we need a separate library?
also if it's needed use strict dependency

Reply:
We need JSON schema validation, not Joi. The reason for that is that we want to integrate this with intellisense in vscode, which only supports JSON schema.

I've tried to look for Joi -> JSON Schema libraries to integrate the `eas.json` validation in https://github.com/expo/vscode-expo, but none would also take a description and other documentation features into account. That's why we opted to go for JSON schema directly.